.github/workflows/update-browser-version.yml

```diff
@@ -24,15 +24,10 @@ jobs:
       - name: Set up Node.js
         uses: actions/setup-node@5e21ff4d9bc1a8cf6de233a3057d20ec6b3fb69d # v3.8.1
         with:
-          node-version: 18
+          node-version: lts/*
           cache: npm
-      - uses: google/wireit@f3a3c79c553122e2fe5829eeac7d815326502903 # setup-github-actions-caching/v1
-      - name: Install and build npm dependencies
-        run: npm ci
-      - name: Download the latest version
-        run: |
-          browser_spec="$(npx @puppeteer/browsers install chrome@canary --path /tmp/browsers | cut -f 1 -d' ')"
-          echo "$browser_spec" > .browser
+      - name: Update browser pin and devtools-protocol
```

Comment:
```suggestion
      - name: Update .browser pin and devtools-protocol
```

Reply:
I disagree as browser pin is `.browser` file. Comment is only valid if you say we should write it as `Update .browser file`.

Reply:
Not a strong opinion here as it was just a nit, feel free to leave as is then